src/ConcreteOperations/linear_combination.jl

```diff
@@ -0,0 +1,29 @@
+export linear_combination
+
+"""
+    linear_combination(P1::SimpleSparsePolynomialZonotope, P2::SimpleSparsePolynomialZonotope)
+
+compute the linear combination of simple sparse polynomial zonotopes `P1` and `P2`.
+
+### Input
+
+- `P1` -- simple sparse polynomial zonotope
+- `P2` -- simple sparse polynomial zonotope
+
+### Output
+
+linear combination of `P1` and `P2`.
+"""
```

Comment:
I think this will be the only implementation of the function in this library, so maybe add the definition of the resulting set here. At least I would not know immediately what `linear_combination` means.

```suggestion
### Output

linear combination of `P1` and `P2`.

### Notes

...
"""
```

Reply:
Of course add something for the `...` :wink:

Reply:
Oh, I didn't notice that 😄 I guess you mean to add the definition of linear combination of two sets, since it's not necessarily known to the user?